.github/workflows/docs-url-alive-check.yaml

```diff
@@ -0,0 +1,61 @@
+on:
+  schedule:
+    - cron:  '0 23 * * 0'
+
+name: Docs URL Alive Check
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.16.x]
+        os: [ubuntu-20.04]
+    runs-on: ${{ matrix.os }}
+    env:
+      TRAVIS: "true"
+      TRAVIS_BRANCH: ${{ github.base_ref }}
+      TRAVIS_PULL_REQUEST_BRANCH: ${{ github.head_ref }}
+      TRAVIS_PULL_REQUEST_SHA : ${{ github.event.pull_request.head.sha }}
+      RUST_BACKTRACE: "1"
+      target_branch: ${{ github.base_ref }}
+    steps:
+    - name: Install Go
+      if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
```

Comment:
I see that you are using this check at every step. Why not just check this once as the first step and return early if true.

Reply:
@Bevisy ^

Reply:
@Bevisy Can you address this so that we can have this PR merged?

Reply:
Ok,on my way.

Reply:
I'm sorry I don't see the syntax for this feature on github action, it looks like the steps can only be executed one at a time in sequence